credsweeper/credentials/line_data.py

```diff
@@ -49,6 +49,9 @@ def __init__(
         self.pattern: re.Pattern = pattern
         # do not store match object due it cannot be pickled with multiprocessing
 
+        # start - end position of matched object
+        self.value_start = -2
```

Comment:
-2 has been chosen to distinguish with standard -1 value when a string was not found.
Anyway, the value is sub-zero and may be skipped if appears.

Reply:
Does -1 has other mean except string was not found?

Reply:
I see.